### PR TITLE
PolyfillQueryService search by feature group aliases and features

### LINKED ISSUE #40: Search for polyfills based on a feature or a feature group/alias

### PR DESCRIPTION
**Pull request by scottmo**
+ Search by features and aliases
+ Tests

Close #40

### CODE REVIEW

File: src/main/java/org/polyfill/interfaces/PolyfillQueryService.java [hunk 1]
```diff
@@ -10,6 +10,5 @@
 // TODO: need to decide the shape later
 public interface PolyfillQueryService {
     List<Polyfill> getPolyfillsByUserAgent(UserAgent userAgent);
-    List<Polyfill> getPolyfillsByAlias(UserAgent userAgent, String alias);
-    List<Polyfill> getPolyfillByFeature(UserAgent userAgent, String name);
+    List<Polyfill> getPolyfillsByFeatures(UserAgent userAgent, String featureNames);
```
Comment (reiniergs) [inline src/main/java/org/polyfill/interfaces/PolyfillQueryService.java]: what if I want multiple features at the same time?

Comment (reiniergs) [inline src/main/java/org/polyfill/interfaces/PolyfillQueryService.java]: I see the features are separated by coma

File: src/main/java/org/polyfill/services/PreSortPolyfillQueryService.java [hunk 2]
```diff
@@ -46,32 +48,37 @@ public void loadConfigs() throws IOException {
...
      * @return the filtered list
      */
-    public List<Polyfill> getPolyfillsByAlias(UserAgent userAgent, String alias) {
-        List<Polyfill> filteredPolyfill = new ArrayList<>();
-        return filteredPolyfill;
+    public List<Polyfill> getPolyfillsByFeatures(UserAgent userAgent, String featureNames) {
+        if (meetsBaseline(userAgent)) {
+            Set<String> featureSet = new HashSet<>();
+
+            List<String> featureNamesList = Arrays.asList(featureNames.split(","));
```
Comment (reiniergs) [inline src/main/java/org/polyfill/services/PreSortPolyfillQueryService.java]: I think we might put this logic(parse the features) outside of the service, It's was confusing to me ... I can pass a list of string instead

Comment (scottmo) [inline src/main/java/org/polyfill/services/PreSortPolyfillQueryService.java]: Yea I thought about that too, but when we receive the URL, the features are in that format, that is, separated by ",". If I handle it here, we don't need to do anything else when using it. Anyways, I can change it to a list.

**Pull request merged**